test/functional/Jsr292/modules/mods.moduleb/module-info.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2018 IBM Corp. and others
+ * Copyright (c) 2017, 2020 IBM Corp. and others
```

Comment:
copyright only change?

Reply:
Restored back to the original copyright for this moved file without any change in the content.